generator/pom.xml

```diff
@@ -65,6 +65,26 @@
 
     <build>
         <plugins>
+            <plugin>
```

Comment:
Unfortunately this doesn't work because there are 2 definitions of the same plugin (exec). Instead, define two execution blocks in the exec plugin found at the bottom of the pom file. 

You must compile the code before invoking exec otherwise the schema preprocessor won't be available. 

Thus you must invoke

```
mvnw compile exec:java@chosenExecIdForPreprocessorExecution
mvnw compile exec:java@chosenExecIdForSchemaGeberator
```

Reply:
Thanks for reviewing the PR @aalmiray 
But I need to invoke `PreprocessSchemas` step before `jsonschema2pojo-maven-plugin`
and `CDEventsGenerator` can only be executed after `jsonschema2pojo-maven-plugin` execution.

Reply:
Ok, phase binding is not working, creating a separate module for `preprocessor` to update schemas.